README.md

```diff
@@ -166,3 +166,34 @@ This project uses `tsup` to wrap the package for `npm`.
 ### Support Maintanance
 If you find this package useful consider a small contribution:
 [Buy Me A Coffee](https://www.buymeacoffee.com/dohomi)
+
+### Contributing
+
+Make sure you're running Node.js 20
+
+1. Clone this repository
+
+```bash
+git clone https://github.com/dohomi/react-hook-form-mui.git
+cd react-hook-form-mui
+```
+
+2. Install dependencies
+
+```bash
+yarn
+```
+
+3. Build the project
```

Comment:
I thought the `prepare` script should be executed instead of built. Remember that `prepare` script is executed automatically after installation if `npm` is used. Yarn does not have such auto script. Instead it could be renamed to `postinstall` in this case it will be executed automatically after `yarn install`